src/components/Datatable/Table/Table.tsx

```diff
@@ -200,6 +209,12 @@ TableProps<D> & { pageButtonsCount?: number }): React.ReactElement {
     collectFetchParams(newPageIndex, pageSize, sortBy);
   };
 
+  const resetHiddenColumns = () => {
+    DatatableStore.update((s) => {
+      s.hiddenColumns = [];
```

Comment:
We could also reset the order here as it feels more like a "take me home" action